app/src/pages/playground/__tests__/playgroundUtils.test.ts

```diff
@@ -82,28 +82,23 @@ const expectedPlaygroundInstanceWithIO: PlaygroundInstance = {
   spanId: "fake-id",
   template: {
     __type: "chat",
-    // These id's are not 0, 1, 2, because we create a playground instance (including messages) at the top of the transformSpanAttributesToPlaygroundInstance function
+    // These id's are not 0 because we create a playground instance (including messages) at the top of the transformSpanAttributesToPlaygroundInstance function
     // Doing so increments the message id counter
     messages: [
-      { id: 2, content: "You are a chatbot", role: "system" },
-      { id: 3, content: "hello?", role: "user" },
+      { id: 1, content: "You are a chatbot", role: "system" },
+      { id: 2, content: "hello?", role: "user" },
     ],
   },
-  output: [{ id: 4, content: "This is an AI Answer", role: "ai" }],
+  output: [{ id: 3, content: "This is an AI Answer", role: "ai" }],
 };
 
 const defaultTemplate = {
   __type: "chat",
   messages: [
     {
       id: 0,
-      role: "system",
-      content: "You are a chatbot",
-    },
-    {
-      id: 1,
       role: "user",
-      content: "{{question}}",
+      content: `How many r's in "{{fruit}}"?`,
```

Comment:
Can we leave this to be answer the question? I find the question variable to be much more generic and useful when switching to dataset mode